### PR TITLE
Added JS unit test runner

### PR DESCRIPTION
**Pull request by PVince81**
Fixes https://github.com/owncloud/activity/issues/360

@nickvergessen 

Note that this is only the ground work. Unit tests to be written separately (at least for newly introduced code!)

Let's see if this runs properly on travis.

### CODE REVIEW

**Comment by PVince81** [comment]
@nickvergessen JS tests can run and pass: https://travis-ci.org/owncloud/activity/jobs/75584373#L406
I think this is ready to merge.

File: .travis.yml [hunk 1]
```diff
@@ -47,6 +47,11 @@ script:
   - sh -c "if [ '$CODECHECK' != '1' -a '$CODECHECK' != '2' -a '$TRAVIS_PHP_VERSION' != 'hhvm' ]; then php ocular.phar code-coverage:upload --format=php-clover clover.xml; fi"
```
Comment (nickvergessen) [inline .travis.yml]: add ` -a '$JSTESTS' != '1'` to all `'$CODECHECK' != '1' -a '$CODECHECK' != '2'`, so we only run either JS or php unit tests so they fail individually :wink:

Comment (PVince81) [inline .travis.yml]: ah yes, good point.

**Comment by PVince81** [comment]
@nickvergessen CI passed, merge ?

**Comment by nickvergessen** [comment]
Tested and can run local :+1: 
Thanks!

**Pull request merged**